http/file_server_test.ts

```diff
@@ -130,7 +130,7 @@ async function fetchExactPath(
   const encoder = new TextEncoder();
   const decoder = new TextDecoder();
   const request = encoder.encode("GET " + path + " HTTP/1.1\r\n\r\n");
-  let conn: void | Deno.Conn;
+  let conn: void | Deno.Conn = undefined;
```

Comment:
The `void` here is strange. I'd definitely use `undefined` here instead. Would it work without assignment with that?

Reply:
Oh, that seems working. Thanks!